[CI] SplitProcessorTests.testSplitWithTargetField fails on processor execution

On 6.x with java 9

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+java9-periodic/192/consoleText

```
  2> REPRODUCE WITH: gradle :modules:ingest-common:test -Dtests.seed=D013609637045D48 -Dtests.class=org.elasticsearch.ingest.common.SplitProcessorTests -Dtests.method="testSplitWithTargetField" -Dtests.security.manager=true -Dtests.jvm.argline="--add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.nio=ALL-UNNAMED --add-opens=java.base/java.nio.file=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.util.concurrent=ALL-UNNAMED --add-opens=java.base/java.util.concurrent.locks=ALL-UNNAMED --add-opens=java.base/java.util.regex=ALL-UNNAMED" -Dtests.locale=ti -Dtests.timezone=CNT
ERROR   0.04s J0 | SplitProcessorTests.testSplitWithTargetField <<< FAILURES!
   > Throwable #1: java.lang.IllegalArgumentException: [wHC] is not an integer, cannot be used as an index as part of path [n.wHC.wQAxOnTsU.GZnVU.򥿤򝶆𫫉󷢘󠬡𱻳󷐲𧟡]
   > 	at __randomizedtesting.SeedInfo.seed([D013609637045D48:31F0D296B0BAD59A]:0)
   > 	at org.elasticsearch.ingest.IngestDocument.setFieldValue(IngestDocument.java:445)
   > 	at org.elasticsearch.ingest.IngestDocument.setFieldValue(IngestDocument.java:403)
   > 	at org.elasticsearch.ingest.common.SplitProcessor.execute(SplitProcessor.java:83)
   > 	at org.elasticsearch.ingest.common.SplitProcessorTests.testSplitWithTargetField(SplitProcessorTests.java:125)
   > 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
   > 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
   > 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   > 	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
   > 	at java.base/java.lang.Thread.run(Thread.java:844)
   > Caused by: java.lang.NumberFormatException: For input string: "wHC"
   > 	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
   > 	at java.base/java.lang.Integer.parseInt(Integer.java:652)
   > 	at java.base/java.lang.Integer.parseInt(Integer.java:770)
   > 	at org.elasticsearch.ingest.IngestDocument.setFieldValue(IngestDocument.java:443)
   > 	... 39 more
```

--- a/modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/SplitProcessorTests.java
+++ b/modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/SplitProcessorTests.java
@@ -120,7 +120,7 @@ public class SplitProcessorTests extends ESTestCase {
     public void testSplitWithTargetField() throws Exception {
         IngestDocument ingestDocument = RandomDocumentPicks.randomIngestDocument(random());
         String fieldName = RandomDocumentPicks.addRandomField(random(), ingestDocument, "127.0.0.1");
-        String targetFieldName = RandomDocumentPicks.randomFieldName(random());
+        String targetFieldName = fieldName + randomAlphaOfLength(5);
         Processor processor = new SplitProcessor(randomAlphaOfLength(10), fieldName, "\\.", false, targetFieldName);
         processor.execute(ingestDocument);
         assertThat(ingestDocument.getFieldValue(targetFieldName, List.class), equalTo(Arrays.asList("127", "0", "0", "1")));

